### PR TITLE
Add clinicalmp discovery workflow

### PR DESCRIPTION
**Pull request by subinamehta**

### CODE REVIEW

**Comment by mvdbeek** [comment]
Just in case you missed it:
Screenshot 2024-11-27 at 17 42 13

**Comment by subinamehta** [comment]
> Just in case you missed it: Screenshot 2024-11-27 at 17 42 13

how do i fix this?

**Comment by mvdbeek** [comment]
You'll have to use smaller datasets or downsample the ones you're using now

**Comment by subinamehta** [comment]
i went from 4 raw files to 1 it still gives me the same error

**Comment by subinamehta** [comment]
Finally!!! Thanks @mvdbeek

File: workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow-tests.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: PTRC_Skubitz_Plex2_F10_9Aug19_Rage_Rep-19-06-08.raw
+        location: https://zenodo.org/records/14182981/files/PTRC_Skubitz_Plex2_F10_9Aug19_Rage_Rep-19-06-08.raw
+  outputs:
+    SGPS MQ Peptides:
+      asserts:
+        - has_n_columns:
```
Comment (mvdbeek) [inline workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow-tests.yml:21]: Could you add some more specific checks here ?

File: workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga [hunk 2]
```diff
@@ -0,0 +1,1359 @@
...
+                "top": 329.70161173161625
+            },
+            "post_job_actions": {
+                "EmailActionout_file1": {
+                    "action_arguments": {
+                        "host": "usegalaxy.eu"
+                    },
+                    "action_type": "EmailAction",
+                    "output_name": "out_file1"
+                },
```
Comment (mvdbeek) [inline workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga:1224]:
```suggestion
```

File: workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga [hunk 3]
```diff
@@ -0,0 +1,1359 @@
...
+                "top": 524.4947117996797
+            },
+            "post_job_actions": {
+                "EmailActionout_file1": {
+                    "action_arguments": {
+                        "host": "usegalaxy.eu"
+                    },
+                    "action_type": "EmailAction",
+                    "output_name": "out_file1"
+                },
```
Comment (mvdbeek) [inline workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga:1057]:
```suggestion
```

File: workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga [hunk 4]
```diff
@@ -0,0 +1,1359 @@
...
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "name": "Clinical Metaproteomics Discovery Workflow",
+    "release": "0.1",
+    "report": {
+        "markdown": "\n# Workflow Execution Report\n\n## Workflow Inputs\n```galaxy\ninvocation_inputs()\n```\n\n## Workflow Outputs\n```galaxy\ninvocation_outputs()\n```\n\n## Workflow\n```galaxy\nworkflow_display()\n```\n"
+    },
+    "steps": {
+        "0": {
+            "annotation": "FastaCLI: MetaNovo Human SwissProt cRAP with decoys on data 1 (MetaNovo Human SwissProt cRAP_concatenated_target_decoy) (49,076 sequences)",
```
Comment (mvdbeek) [inline workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga:21]: This seems oddly specific as a description. Can you state what this input is, and maybe where to get it from if it is not obvious to users ?

File: workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow-tests.yml (hunk 1 above)
Comment (subinamehta) [reply workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow-tests.yml:21]: The output can be different based on databases being added, so I thought there should be a column at least. Should I specify rows?

File: workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga [hunk 5]
```diff
@@ -0,0 +1,1345 @@
...
```
Comment (mvdbeek) [inline workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga:21]:
```suggestion
            "annotation": "",
```

File: workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga [hunk 6]
```diff
@@ -0,0 +1,1345 @@
...
+            },
+            "tool_state": "{\"source\": {\"from\": \"cRAP\", \"__current_case__\": 1}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "0.3.4",
+            "type": "tool",
+            "uuid": "6f0be889-015d-4be0-a997-dd63e18344eb",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "5": {
+            "annotation": "Experimental Design Discovery MaxQuant",
```
Comment (mvdbeek) [inline workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow.ga:199]: Here you want to describe what is in that file and where to get it from.

**Review by mvdbeek — APPROVED** [review]

**Pull request merged**